mlflow/server/js/.eslintrc.js

```diff
@@ -129,7 +129,7 @@ module.exports = {
     }],
     'max-lines': [2, 1000],
     'max-nested-callbacks': 2,
-    'max-params': [2, 8],
+    'max-params': [2, 12],
```

Comment:
Needed because the number of params passed to `Routes.getMetricPageRoute` is now quite large

Reply:
SGTM, will make the change